pkg/event_handler/consumer/consumer_test.go

```diff
@@ -0,0 +1,106 @@
+package consumer
+
+import (
+	"github.com/confluentinc/confluent-kafka-go/kafka"
+	"testing"
+	"time"
+
+	"github.com/port-labs/port-k8s-exporter/pkg/config"
+)
+
+type Fixture struct {
+	t            *testing.T
+	mockConsumer *MockConsumer
+	consumer     *Consumer
+	producer     *kafka.Producer
+	topic        string
+}
+
+type MockConsumer struct {
+	pollData kafka.Event
```

Comment:
Check the tests logs, it tries to connect to Kafka and failing & the for loop is creating a lot of new event logs